src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/main.js

```diff
@@ -170,8 +170,20 @@ define([
                     }
                 );
             } else {
-                this.data = this.content.toJSON();
-                promise.resolve();
+                var baseContent = new Content();
+                baseContent.fullFetch(
```

Comment:
So we have to send a request when creating a new page? Where does this request go? I think that's a little bit much, especially since the permissions for the current user can be taken from `config.get('sulu_security.contexts')`.

Reply:
Thanks for the tip! I'll change this to use `sulu_security.contexts` instead.